app/controllers/courses/sites_controller.rb

```diff
@@ -13,7 +13,7 @@ def update
       selected_site_ids = params.dig(:course, :site_statuses_attributes)
         .values
         .select { |f| f["selected"] == "1" }
-        .map { |f| f["id"].to_i }
+        .map { |f| f["id"] }
```

Comment:
Ok, now I'm confused. You _have_ to leave them as strings? So this means that object ids are being returned as strings for these JSONAPI resources?

Reply:
Y E S

Reply:
I just had to see for myself.

```
I, [2019-06-05T16:48:43.960678 #75522]  INFO -- omniauth: (dfe) Request phase initiated.
I, [2019-06-05T16:49:07.534751 #75522]  INFO -- omniauth: (dfe) Callback phase initiated.

From: /Users/misaka/Code/DfE/manage-courses-frontend/app/controllers/providers_controller.rb @ line 13 ProvidersController#show:

    10: def show
    11:   @provider = Provider.find(params[:code]).first
    12:   binding.pry
 => 13: end

[1] pry(#<ProvidersController>)> @provider.id
=> "12035"
```

Ok, not what I expected.

Reply:
I wonder if this is something we should be deserialising by hand.